lib/3scale/backend/transactor/status.rb

```diff
@@ -200,31 +200,27 @@ def add_plan_section(xml, tag, plan_name)
         end
 
         def add_authorize_section(xml)
-          xml << '<authorized>'
           if authorized?
-            xml << 'true' \
-                   '</authorized>'
+            xml << '<authorized>true</authorized>'.freeze
           else
-            xml << 'false' \
-                   '</authorized>' \
-                   '<reason>' \
-                   "#{rejection_reason_text}" \
-                   '</reason>'
+            xml << '<authorized>false</authorized><reason>'.freeze
+            xml << rejection_reason_text
+            xml << '</reason>'.freeze
           end
         end
 
         def add_application_section(xml)
           redirect_uri = application.redirect_url
-          xml << '<application>' \
+          xml << "<application>" \
```

Comment:
I'd say this was correctly using single quotes. There is no interpolation within them so it signals to the reader: don't worry, this is the actual string, no interpolation there.

Reply:
:+1: changed.